js-core/lesson1.md

```diff
@@ -4,198 +4,428 @@
+>
+> Refer to the three variables that you just created to get their types (i.e. `console.log(typeof courseName)` etc...)
+>
+> Is your output the same?
+
+
+# If/Else Statements
+
+Now we're going to start adding in some **logic**. So far we have been logging the same thing every time we run the code, but what if we want to log according to different conditions? In programming there is something called an "if/else statement". It tests conditions, and will perform different actions based on the outcome of these tests.
+
+The structure of an if/else statement in JavaScript is as follows:
 
 ```js
-var countries = ['Scotland', 'Germany', 'Syria'];
+if (condition) {
+  // do something
+} else {
+  // do something else
+}
+```
+
+It basically translates to "if this condition is true, do this - or else do something else".
+
+### Try it out
+
+What if we want to `console.log()` "Hello world!" if we're feeling happy, and something else if we're sad?
 
-countries[1] = 'Sudan';
+Let's define a variable called `isHappy`, and assign it a boolean value of `true` or `false` (depending on how happy you're feeling).
 
-console.log(countries); // ['Scotland', 'Sudan', 'Syria']
+```js
+var isHappy = true;
 ```
 
-You can also create new array elements by just writing to the respective index.
+Then let's write our if/else statement. In our condition, we want to test if the variable `isHappy` is equal to `true`. Note that in JavaScript, to test if two things are equal you need to use a triple equals (`===`) rather than just one (`=`).
 
 ```js
-var countries = ['Scotland', 'Germany', 'Syria'];
+if (isHappy === true) {
+  // do something
+} else {
+  // do something else
+}
+```
 
-countries[3] = 'Sudan';
+Now let's add in what will happen if the conditions are met. The comments are there to explain what is happening.
 
-console.log(countries); // ['Scotland', 'Germany', 'Syria', 'Sudan']
+```js
+if (isHappy === true) {
+  // if I am happy, print "Hello world!"
+  console.log('Hello world!');
+} else {
+  // if I am sad, print a frowny face
+  console.log(':(');
+}
 ```
 
-You can also use other variables to store in an array, and integer variables to define the index you want to read from.
+Now run this code (not forgetting to put the variable at the top of the Repl), and see what happens! Try changing the value of your variable from `true` to `false`.
+
+> **Exercise**
+>
+> Write an if/else statement that evaluates if a number is even or odd. If it is even, it will print the string `'even'`, and if it is odd, it will print `'odd'`.
+>
+> For this challenge, you might find it useful to use the "modulo" operator, which looks like this: `%`. The modulo operator finds the remainder after division of one number by another. For example:
+>
+> ```js
+>13 % 2 === 1 // 13 divided by 2 leaves a remainder of 1
+>100 % 10 === 0 // 100 divided by 10 leaves a remainder of 0
+>```
+
```

Comment:
I'll try to think of another exercise we can add here and submit a PR.  I suspect this whole lesson will be hard for people who've never done programming before, but very easy and hopefully not too boring for people who've programmed before. So I'll see if I can add some more advanced exercises here and there for students who finish early.